_docs/4-kubernetes/0-intro.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Introduction
+permalink: /docs/kubernetes-integration-intro.html
+redirect_from: /docs/kubernetes-integration/
+category: Kubernetes Integration
+order: 0
+---
+
+
+When deployed on [Kubernetes](https://kubernetes.io), Amalgam8 provides a near native experience
+ by reusing native Kubernetes tools and fucntions to run and manage its control plane components.
+ The [Service Registry](/docs/control-plane-registry.html) functionality leverages Kubernetes
+ services, and the [Route Controller](/docs/control-plane-controller.html) uses
+ Kubernetes [Third Party Resources](https://kubernetes.io/docs/user-guide/thirdpartyresources/)
+ to store routing rules, thus enabling familiar access using `kubectl` command line.
+
+*__Before__* begining working through the examples and demos, pelase confirm all container images for the control plane,
+ [helloworld](docs/demo-helloworld.html) and [bookinfo](/docs/demo-bookinfo.html) demo applications, are stored in a
+ registry accessible to the cluster.
+ By default, the latest images are hosted in the `amalgam8` docker hub image repository, but if you're
+ building a different version, be sure to push the modified images and change the resource manifests accordingly.
+
+### Sidecar Configuration <a id="sidecar-config"></a>
+
+The Amalgam8 sidecar configuration need to change to match the fact that it is running in
+ in a Kubernetes cluster. Please refer also to [sidecar configuration](/docs/sidecar-configuration.html).
+
+ - The `discovery_adapter` and `rules_adapter` flags should be set to `kubernetes`
+ - Service instance registration is not required, pods are automatically registered by Kubernetes.
+ - The access URLs and token are not required (they are automatically retrieved from the pod).
+  The default, in-cluster, configuration may be overridden by setting the `kubernetes_url` and
+  `kubernetes_token` flags.
+ - The sidecar will fetch rules and service instance information from the `default` namespace.
+   This may be overridden by setting the `kubernetes_namespace` flag.
+ - If the `kubernetes_pod_name` is set, the source service name and tags are automatically
```

Comment:
No, set. If the value is defined we will automatically pull in the info based on that. It's not something that a container in a pod has access to.